runtime/long.go

```diff
@@ -321,6 +321,7 @@ func initLongType(dict map[string]*Object) {
 	LongType.slots.New = &newSlot{longNew}
 	LongType.slots.NonZero = longUnaryBoolOpSlot(longNonZero)
 	LongType.slots.Or = longBinaryOpSlot(longOr)
+	LongType.slots.Pow = &binaryOpSlot{longPow}
```

Comment:
Is there a reason not to use longBinaryOpSlot() here? It's a well established pattern here and it takes care of a little bit of the type coercion for you.